.github/actions/localize-push-pull/entrypoint.sh

```diff
@@ -51,9 +67,16 @@ if [ "$ACTION" = "push" ]; then
 elif [ "$ACTION" = "pull" ]; then
   create_config
   localize pull
-  cp $GITHUB_WORKSPACE/$INPUTPATH/en.json $GITHUB_WORKSPACE/$OUTPUTPATH/en.json
-  if [ "$RESTRUCTURE" = true ]; then
+  if [ "$INPUTPATH" != "$OUTPUTPATH" ]; then
+    cp -p $GITHUB_WORKSPACE/$INPUTPATH/en.json $GITHUB_WORKSPACE/$OUTPUTPATH/en.json
+  fi
+  if [ "$RESTRUCTURE" = "true" ]; then
     restructure_files
+  elif [ "$ALPHABETIZE" = "true" ]; then
+    # alpha for non-restructured output
+    for lang in $LANGUAGES_ARR; do
```

Comment:
this would not alphabetize the en/translation.json file. Should we add a check that if `ALPHABETIZE = true && RESTRUCTURE = true` also perform the action for `en/translation.json`

Reply:
nvm i think your logic does that already

Reply:
We're already sorting the restructured output, but we're explicitly _not_ sorting the _input_ in any case, which is intentional here. Does that make sense?